portal.go

```diff
@@ -453,12 +453,7 @@ func (portal *Portal) markRead(intent *appservice.IntentAPI, eventID id.EventID,
 		return nil
 	}
 
-	// Only send the read update if it is more recent than the last read time
-	if portal.ReadAt.After(readAt) {
-		return nil
-	} else {
-		portal.ReadAt = readAt
-	}
+	portal.ReadAt = readAt
```

Comment:
the ReadAt field should be removed entirely

Reply:
just removed it in the next commit

Reply:
Thank you

Reply:
Now?